Sources/TokamakCore/CGStubs.swift

```diff
@@ -15,7 +15,11 @@
 //  Created by Max Desiatov on 08/04/2020.
 //
 
+#if canImport(Glibc)
 import Glibc
+#else
+import Foundation
+#endif
```

Comment:
Yeah, `import Darwin` could work better:
```suggestion
#elseif canImport(Darwin)
import Darwin
#endif
```